core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -67,6 +67,23 @@ class SparkContextSuite extends FunSuite with LocalSparkContext {
     }
   }
 
+  test("Test getOrCreateContext") {
```

Comment:
That's technically not what you called the method! Can you just make this `test("getOrCreate")`